src/runtime/contrib/cublas/cublas.cc

```diff
@@ -342,13 +434,34 @@ TVM_REGISTER_GLOBAL("tvm.contrib.cublas.matmul")
     }
 });
 
+TVM_REGISTER_GLOBAL("tvm.contrib.cublaslt.matmul")
```

Comment:
I think this registration should be guarded with if CUDART_VERSION >= 10010, rather then giving runtime error at L455

Reply:
Sure, I'll update this.